cluster/examples/kubernetes/ceph/csi/rbd/storageclass-test.yaml

```diff
@@ -7,6 +7,12 @@ spec:
   failureDomain: host
   replicated:
     size: 1
+    # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
+    # for more info: https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size
+    #targetSizeRatio: .5
+    # Allow setting pool with replica 1, this could lead to data loss without recovery.
+    # Make sure you're *ABSOLUTELY CERTAIN* that is what you want
+    #allowUnsafeSingleReplica: false
```

Comment:
The intention of the examples that end in `-test.yaml` is that they will work for replica 1. So we would need to set this to true.